signal/signal.go

```diff
@@ -93,6 +98,7 @@ func mainInterruptHandler() {
 		case <-quit:
 			log.Infof("Gracefully shutting down.")
 			close(shutdownChannel)
+			atomic.StoreInt32(&started, 0)
```

Comment:
could add this to a defer in the beginning of the method.

Reply:
Oh yeah that definitely makes sense.